tests/fixtures/test_GFM_style.html

```diff
@@ -0,0 +1 @@
+<img src="https://example.com/badge.png" style="width: 20px; height: 20px;">
```

Comment:
Sorry I hadn't looked at the code changes before, this isn't going to work on PyPI, we don't allow inline styles via CSP, so anything that relies on the ``style`` attribute is not going to work, and I would be a hard -1 on relaxing our CSP rules to allow it.